### PR TITLE
add a delay between sending commands using separate_turn_on_commands

### LINKED ISSUE #383: Brightness and Color not updated reliably by bulb when using separate_turn_on_commands
#### Version information:
1.2.1

#### Description:

My bulbs cannot handle brightness and color in one command. Thus, I'm using the separate_turn_on_commands feature with transition. This works fine in general, but the bulb doesn't update both reliably. I think the reason for this is the quite congested 2,4-GHz which leads fluctuating delays in the zigbee network.

If the first command has a higher delay than the second, the first command isn't apply completly when the seconds command (with a low delay) arrives. Then the seconds command overrides the first. Therefore, the brightness isn't at the targeted level. A small delay between the two commands may solve the issue.

### PR DESCRIPTION
**Pull request by Mark-Niemeyer**
closes #383

a delay of 100 ms solves my problems reliably

### CODE REVIEW

**Comment by basnijholt** [comment]
@all-contributors
please add @Mark-Niemeyer for code

**Comment by basnijholt** [comment]
Thanks a lot!

Mind also adding a description to the table in the README?

File: custom_components/adaptive_lighting/switch.py [hunk 1]
```diff
@@ -872,7 +874,7 @@ async def turn_on(service_data):
             if len(service_datas) == 2:
                 transition = service_datas[0].get(ATTR_TRANSITION)
                 if transition is not None:
-                    await asyncio.sleep(transition)
+                    await asyncio.sleep(transition + self._send_split_delay / 1000.0)
```
Comment (basnijholt) [inline custom_components/adaptive_lighting/switch.py:877]: If there is no transition, the send_split_delay will be ignored.

Comment (Mark-Niemeyer) [reply custom_components/adaptive_lighting/switch.py]: Good remark. I moved it out of the if

**Comment by basnijholt** [comment]
Awesome, thanks a lot! If you fix the merge conflict, I can merge it right away. Otherwise, I’m back to my computer tomorrow and I can fix it.

**Comment by basnijholt** [comment]
🎉 

Thanks again!

**Pull request merged**